app/components/Button.tsx

```diff
@@ -4,13 +4,24 @@ interface ButtonProps {
   size: 'small' | 'medium' | 'large';
   label: string;
   varient: 'primary' | 'secondary';
+  width: 'w-full' | 'w-auto';
```

Comment:
If we set `w-full` by default and put this button inside a container will it not remove the need for this prop to be passed in as it will take the container width anyway?

Reply:
makes sense